AzureResourceManager/JCR/azuredeploy.parameters.json

```diff
@@ -21,7 +21,7 @@
       "value": "GEN-UNIQUE"
     },
     "masterKey": {
-      "value": "35767fa0164bac66b6cccb8880babefb"
+      "value": "GEN-UNIQUE"
```

Comment:
Does it generate master key that is compatible with artifactory? I assume it does but I am surprised.